src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int length;
```

Comment:
It's better to name this field 'height' instead

Reply:
++